pkgs/desktops/gnome-3/misc/gnome-screensaver/gnome-xscreensaver/setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup
+
+setup(
+  name='gnome-xscreensaver',
+  version='0.1.0',
+  py_modules=['gnomexscreensaver'],
+  install_requires=['pygobject'],
+  entry_points='''
+      [console_scripts]
+      gnome-screensaver=gnomexscreensaver:main
```

Comment:
```suggestion
      gnome-xscreensaver=gnomexscreensaver:main
```

Reply:
I want this to be a drop-in replacement for `gnome-screensaver`, hence the executable name.

Reply:
I would expect that to be confusing in process manager. Or is there something that attempts to run it directly?